core/arch/aarch64/proc.c

```diff
@@ -34,17 +34,35 @@
 #include "proc.h"
 #include "instr.h"
 
+// The whitepaper below documents AArch64 words being 32 bits wide.
+// https://developer.arm.com/-/media/Files/pdf/
+// graphics-and-multimedia/Porting%20to%20ARM%2064-bit.pdf
+#define AARCH64_WORD_SIZE_BYTES 4
+
 static int num_simd_saved;
 static int num_simd_registers;
 static int num_opmask_registers;
 
+void
```

Comment:
As Assad pointed out: this should be static, and should probably have a note that it will only work at init time (it will crash later trying to write to .data)

Reply:
However, this will likely go away and be replaced with sthg like an assignment to the var inside the init function to a call to a drlibc function.

Reply:
Removed this code and using the drlibc function now.